Directory.Build.props

```diff
@@ -429,6 +429,11 @@
     <PublishWindowsPdb>false</PublishWindowsPdb>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(UsingMicrosoftNoTargetsSdk)' == 'true'">
+    <!-- Explicitly disable running analyzers to avoid trying to discover the correct ILLink tool pack for a NoTargets project. -->
+    <RunAnalyzers>false</RunAnalyzers>
+  </PropertyGroup>
```

Comment:
nit: can you please move this into Analyzers.targets? We already have some conditions there to determine when to not run analyzers.